web/src/beta/features/PluginPlayground/Code/hook.ts

```diff
@@ -51,10 +51,69 @@
 export default ({ files }: Props) => {
   const [widgets, setWidgets] = useState<Widgets>();
   const [, setNotification] = useNotification();
+  const [fileOutputs, setFileOutputs] = useState<
+    {
+      title: string;
+      output: string;
+    }[]
+  >();
 
   const executeCode = useCallback(() => {
     const ymlFile = files.find((file) => file.title.endsWith(".yml"));
 
+    const jsFiles = files.filter((file) => file.title.endsWith(".js"));
+
+    const outputs = jsFiles.map((file) => {
+      const fn = new Function(
+        `"use strict";
+        const reearth = {
+          ui: {
+        show: function () {}
+          },
+          popup: {
+        show: function () {}
+          },
+        modal: {
+        show: function () {}
+          
+        };
```

Comment:
This time, I have added mock definitions to prevent undefined errors at runtime. Therefore, it is not necessary to include the actual behavior. If there is a better approach, I would appreciate your guidance.